frontend/cert/adhoc_provider.go

```diff
@@ -44,6 +44,7 @@ func (provider *AdhocProvider) GetExistingCertificate(
 	serverName name.ServerName,
 ) (*tls.Certificate, error) {
 	cache, _ := provider.cache.Load().(certificateCache)
+
```

Comment:
Just FYI if you change a lot of unrelated things like this it makes it really hard to review PRs.

Reply:
I was just clearing some linter warnings, housekeeping I added was switching to wrapped errors and removing `context.Context` from `loadCertificate` since it wasn't used.

Reply:
I was just clearing some linter warnings, housekeeping I added was switching to wrapped errors and removing context.Context from loadCertificate since it wasn't used.